### PR TITLE
CNV-22041: RN for VM export manifests

### PR DESCRIPTION
**Pull request by ousleyp**
Version(s): 4.13 only/no CP

Issue: [CNV-22041](https://issues.redhat.com//browse/CNV-22041)

Link to docs preview: http://file.rdu.redhat.com/pousley/CNV-22041/virt/virt-4-13-release-notes.html#virt-4-13-new

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: virt/virt-4-13-release-notes.adoc [hunk 1]
```diff
@@ -59,6 +59,7 @@ All VM templates that are included with {VirtProductName} now use this machine t
 //** To learn more, see xref .. virt/upgrading-virt.adoc #virt-rhel-9_upgrading-virt[{VirtProductName} on {op-system-base} 9].
 
 //CNV-22041 Release note: NEW Retrieve a portable VM definition (yaml) when exporting a VM or VMSnapshot
+* You can now obtain the `VirtualMachine`, `DataVolume`, and `ConfigMap` manifests from the export server after you export a VM or snapshot. Learn more about xref:../virt/virtual_machines/virt-exporting-vms.adoc#virt-accessing-exported-vm-manifests_virt-exporting-vms[accessing exported VM manifests].
```
Comment (bergerhoffer) [inline virt/virt-4-13-release-notes.adoc:62]: Totally fine if you use this wording often other places, but the "Learn more about" sounds slightly off to me. I think people more often do the "For more information, see <link>."  wording

**Review by bergerhoffer — COMMENTED** [review]
Minor suggestion, but LGTM!

File: virt/virt-4-13-release-notes.adoc [hunk 2]
```diff
@@ -59,6 +59,7 @@ All VM templates that are included with {VirtProductName} now use this machine t
 //** To learn more, see xref .. virt/upgrading-virt.adoc #virt-rhel-9_upgrading-virt[{VirtProductName} on {op-system-base} 9].
 
 //CNV-22041 Release note: NEW Retrieve a portable VM definition (yaml) when exporting a VM or VMSnapshot
+* You can now obtain the `VirtualMachine`, `DataVolume`, and `ConfigMap` manifests from the export server after you export a VM or snapshot. For more information, see xref:../virt/virtual_machines/virt-exporting-vms.adoc#virt-accessing-exported-vm-manifests_virt-exporting-vms[accessing exported VM manifests].
```
Comment (jpeimer) [inline virt/virt-4-13-release-notes.adoc:62]:
```suggestion
* You can now obtain the `VirtualMachine`, `ConfigMap`, and `Secret` manifests from the export server after you export a VM or snapshot. For more information, see xref:../virt/virtual_machines/virt-exporting-vms.adoc#virt-accessing-exported-vm-manifests_virt-exporting-vms[accessing exported VM manifests].
```

**Review by jpeimer — CHANGES_REQUESTED** [review]

**Review by jpeimer — APPROVED** [review]
Thanks, Pan!
/lgtm

**Pull request merged**